service/host_mock.go

```diff
@@ -45,9 +50,16 @@ func NopHostAuthFailure() Host {
 
 // NopHostConfigured is a nop host with set logger and tracer for tests
 func NopHostConfigured(client auth.Client, logger ulog.Log, tracer opentracing.Tracer) Host {
+	return nopHostConfigured(client, logger, tracer, nil)
+}
+
+func nopHostConfigured(client auth.Client, logger ulog.Log, tracer opentracing.Tracer, configProvider config.Provider) Host {
+	if configProvider == nil {
+		configProvider = config.NewStaticProvider(nil)
+	}
 	return &serviceCore{
 		authClient:     client,
-		configProvider: config.NewStaticProvider(nil),
+		configProvider: configProvider,
 		standardConfig: serviceConfig{
 			Name:        "dummy",
```

Comment:
can we read the name from our config provider? i need this for some integration tests.